lib/phoenix_live_view/helpers.ex

```diff
@@ -172,6 +172,40 @@ defmodule Phoenix.LiveView.Helpers do
   opposed to having many modules with a single `render/1` function. Function
   components support other important features, such as slots. You can learn
   more about components in `Phoenix.Component`.
+
+  ### HEEx extension: special attributes
+
+  Apart from normal HTML attributes, HEEx also support some special attributes
+  such as `:for` and `:let`.
+
+  #### :for
+
+  It is a syntax sugar for `<%= for .. do %>` that can used only in normal HTML
+  tags. Therefore `:for` will not work on components.
+
+  ```eex
+  <table id="my-table">
+    <tr :for={user <- @users}>
+      <td><%= user.name %>
+    </tr>
+  <table>
+  ```
+
+  The snippet above will generate a list of `td` as you would expect.
+
+  #### :let
+
+  Useful for binding variables that will be used by the children elements.
+
+  ```eex
+  <.form :let={f} for={@changeset} phx-change="validate" phx-submit="save">
+    <%= label f, :username %>
+    <%= text_input f, :username %>
```

Comment:
```suggestion
    <%= label(f, :username) %>
    <%= text_input(f, :username) %>
```

That's how it gets formatted, I think it's worth using the same pattern on docs.